exporter.json

```diff
@@ -1,25 +1,21 @@
 {
-    "id": "io.supernova.react",
-    "name": "React",
-    "description": "React token and style exporter",
-    "source_dir": "src",
-    "version": "1.2.2",
-    "tags": [
-        "React",
-        "Tokens",
-        "Styles"
-    ],
-    "author": "Ivan Terpugov & Jiri Trecak",
-    "organization": "Supernova",
-    "homepage": "https://supernova.io",
-    "usesBrands": true,
-    "config": {
-        "output": "output.json",
-        "sources": "sources.json",
-        "js": "src/js/helpers.js"
-    },
-    "engines": {
-        "pulsar": "1.0.0",
-        "supernova": "1.0.0"
-    }
+  "id": "io.acorns.react",
```

Comment:
In the docs, it says this about the exporter ID:
> If you don't plan to make your modification public, feel free to skip this step.

What does that mean for us?

Reply:
Also, it says:
> Please note that all exporter package repositories must be public, for now. We are currently working on a way to link private repositories as well, so you can create exporters for proprietary technologies you don't want others to know. Stay updated throughout the duration of Early Access by joining our [Discord community server](https://community.supernova.io/).

So, I'm worried that if we have to include proprietary technologies and code, then all code included in this and our other exporter repositories would be licensed officially as open source.

Reply:
Based on the docs, it looks like we opt in to making the exporter available to others

> this will save you some time later when you want to make it available to other users

Reply:
So yeah updating the id is not necessary but it makes sense to me to update it for our own reference

Reply:
I am not to worried about the nature of the code we are including within the repositories themselves but that's just based on what I know needs to be done within the Web client